src/frontend/src/pages/Login.tsx

```diff
@@ -12,38 +12,47 @@ import {
 import PersonIcon from '@mui/icons-material/Person';
 import axios from 'axios';
 import { useState } from 'react';
-
-function sendData(user: string, password: string) {
-   axios({
-      method: 'post',
-      url: '/login',
-      data: {
-         user: user,
-         password: password,
-      },
-   })
-      .then((response) => {
-         console.log(response);
-      })
-      .catch((error) => {
-         if (error.response) {
-            console.log(error.response);
-            console.log(error.response.status);
-            console.log(error.response.headers);
-         }
-      });
-}
+import { useNavigate } from 'react-router-dom';
 
 export default function Login() {
+   const navigate = useNavigate();
    const [user, setUser] = useState('');
    const [password, setPassword] = useState('');
+   const [res, setRes] = useState();
+
+   function sendData() {
+      axios({
+         method: 'post',
+         url: 'http://127.0.0.1:5000/login',
+         withCredentials: true,
+         data: {
+            user: user,
+            password: password,
+         },
+      })
+         .then((response) => {
+            console.log(response);
+            setRes(response.data);
+         })
+         .catch((error) => {
+            if (error.response) {
+               console.log(error.response);
+               console.log(error.response.status);
+               console.log(error.response.headers);
+               setRes(error.response.data);
+            }
+         });
+   }
 
    const paperStyle = {
       padding: 20,
-      height: '55vh',
+      height: '60vh',
       width: 310,
       margin: '20px auto',
    };
+   if (res === `welcome ${user}`) {
+      navigate('/');
+   }
```

Comment:
I'm being redirected anytime I press the sign in button

Reply:
wait maybe not

Reply:
Works on my machine lol

Reply:
nvm from that. Anyway, there's an error in the console that's related to this

Reply:
> nvm from that. Anyway, there's an error in the console that's related to this

Whats the error

Reply:
![Screenshot 2022-11-29 at 12 27 00 AM](https://user-images.githubusercontent.com/59807670/204455065-38ac19dd-7d9e-44c9-af46-c6087705cbbb.png)
From what I know, you should look at the first non-react trace which in this case is Login.tsx line 54

Reply:
Just pushed some commits. Try if the warnings still persist.